core/src/test/java/org/jsmart/zerocode/core/kafka/helper/KafkaConsumerHelperTest.java

```diff
@@ -160,21 +160,23 @@ public void should_read_json_with_headers_in_record() throws IOException {
 
     @Test
     public void test_firstPoll_exits_early_on_assignment() {
+        consumerCommon = new ConsumerCommonConfigs(true, false, "aTestFile", "JSON", true, 3, 5000L, "");
```

Comment:
Style thing: as this belongs to the given part of the test I would move this line beneath the comment.

Reply:
@ddingel addressed your comment.